jetty-nosql/src/main/java/org/eclipse/jetty/nosql/mongodb/MongoSessionDataStore.java

```diff
@@ -269,7 +269,7 @@ public SessionData doLoad(String id) throws Exception
     public boolean delete(String id) throws Exception
     {
         if (LOG.isDebugEnabled())
-            LOG.debug("Remove:session {} for context {}", id, _context);
+            LOG.debug("Remove:session {} for context ", id, _context);
```

Comment:
That has mismatched parameters now.

Reply:
I don't know what's happening here, but it's certainly correct in jetty -10.0.x repo.